.github/compat-data-helper/package-lock.json

```diff
@@ -0,0 +1,8193 @@
+{
+  "name": "bcd-bot",
+  "version": "1.0.0",
```

Comment:
```suggestion
  "version": "0.1.0",
```

Reply:
This supposes a level of confidence in my work that I myself do not have 😆

Reply:
Lol, didn't mean to lower your work to a .0 version or anything, but this is what you have in package.json, so it needs to match here in package-lock :smile:

Reply:
Ah, you changed it in package.json to 1.0.0. That works for me :)